assets/containerisableComponents/marketingConsent/marketingConsent.scss

```diff
@@ -44,4 +62,16 @@
       left: 321px;
     }
   }
-}
\ No newline at end of file
+}
+
+.component-marketing-consent__confirmation-message {
+  .component-dotcom-cta {
+    margin-top: $gu-v-spacing * 2;
+  }
```

Comment:
Could this go inside `.component-page-section--marketing-consent` instead, so that there's only one place where we're scoping the marketing consent components with nesting?

Reply:
Unfourtanely no, this is because dotcom cta has its own PageSection and everything is wrapped as a different component.